### ISSUE TITLE
Template containing html is loaded as empty

### ISSUE THREAD

**Issue by lcsqlpete** [bobthecow/mustache.php#368]
I'm having a strange problem with one of the many templates I'm using.

The bottom line is that the template and the array I pass to the render function results in an empty string.

I inserted a couple of debug statement in the FilesystemLoader.php script in the load function to display the the content of $this->templates[$name] and I can see that an empty string is loaded instead of the template contents.  The $name parameter contains the correct file name.

The name of the template file is "users_table", it has the normal .mustache suffix and is in the correct template directory. I've attached it below.
[users_table.mustache.zip](https://github.com/bobthecow/mustache.php/files/5947023/users_table.mustache.zip)

I'm at a loss to figure out what's causing this problem so grateful for any suggestions.

Pete

**Comment by bobthecow** [comment]
I don't see anything in this file that would result in it not loading. 

What happens if you replace the template content with static text? Does it load and render properly?

**Comment by lcsqlpete** [comment]
Tried that and still get the same problem. I very much doubt this is a Mustache problem because all my other templates are working correctly, just wish I could figure out why it's happening.

**Comment by lcsqlpete** [comment]
OK, I figured this out - the file permissions were set incorrectly, allowing read access to User only.  Once I fixed that, all works OK.

Not sure if it's possible but would be good if a permissions errors could be caught.

**Comment by bobthecow** [comment]
It might be possible. Do you know what specifically they were set to?

**Comment by lcsqlpete** [comment]
Yes, they were set to #600 when I was getting the error and I changed them to #644 to fix the problem.  I don't know enough about website file permissions to say that #644 is the only correct setting, I just used that because all the other templates were set that way.
